### DISCUSSION TITLE
New release: "Ellie" 2023.11.5

### DISCUSSION THREAD

**Discussion by pschuele** [tine-groupware/tine#21] [Announcements]
We just released the new #tinegroupware version "Ellie" 2023.11.5 🎉

Check https://tine-groupware.de, https://hub.docker.com/r/tinegroupware/tine and https://github.com/tine-groupware/tine/releases/tag/2023.11.5 for more information and the downloads.

**Comment by lab-at-nohl** [comment]
Your versionCheck still reports Elli 2023.11.3.

**Reply by pschuele** [reply]
yes, we currently have a problem with the packages on https://packages.tine20.com/maintenance/source/ ... we'll update the versioncheck when they are available

**Reply by pschuele** [reply]
the packages are now available, the versioncheck will be updated very soon

**Reply by pschuele** [reply]
versioncheck is also updated
